WordPress/src/main/res/values/strings.xml

```diff
@@ -1086,9 +1086,9 @@
     <string name="scan_fixing_threats_description">We\'re hard at work fixing these threats in the background. In the meantime feel free to continue to use your site as normal, you can check the progress at anytime.</string>
     <string name="scan_scanning_is_initial_description">Welcome to Jetpack Scan, we are taking a first look at your site now and the results will be with you soon.</string>
     <string name="scan_provisioning_description">Welcome to Jetpack Scan! We\'re scoping out your site, setting up to do a full scan. We\'ll let you know if we spot any issues that might impact a scan, then your first full scan will start.</string>
-    <string name="scan_idle_manual_scan_description">Run a manual scan now or wait for Jetpack to scan your site later today.</string>
-    <string name="scan_idle_last_scan_description">The last Jetpack scan ran %s and everything looked great. %s</string>
-    <string name="scan_idle_with_threats_description">The scan found %1s potential threats on %2s. Please review them below and take action. We\'re %3$s if you need us.</string>
+    <string name="scan_idle_manual_scan_description">To review your site again run a manual scan, or wait for Jetpack to scan your site later today.</string>
+    <string name="scan_idle_last_scan_description">The last Jetpack scan ran %s and did not find any risks. %s</string>
```

Comment:
Question (❓): Should `ran %s` be included? It is not mentioned in the copy improvements.

Reply:
I kept is similar to iOS copy here: https://github.com/wordpress-mobile/WordPress-iOS/pull/15790/files#diff-2e82f404f18074738a562916fc83c38dfff2ae22135da879c71c8d19e5703693R207

@osullivanchris, let us know if we should remove `ran %s` from this label.

Reply:
Sorry have to write a bit here to make sure that we are all talking about the same thing. I got editorial reviews of three scenarios:

### 1 Scan with Threats
**Unchanged**
Heading: Your site may be at risk
Body: The scan found 2 potential threats with Brothers BBQ. Please review them below and take action. We are here to help if you need us.

### 2 Scan without threats

**Before**
Heading: Don't Worry About a Thing
Subheading: The last jetpack scan ran just now and everything looked great. Run a manual scan now or wait for Jetpack to scan your site later today.
**After**
Heading: No Threats Found
The last jetpack scan did not find any risks. To review your site again run a manual scan, or wait for Jetpack to scan your site later today.

### 3 Scan Just Fixed

**Before**
You resolved 2 threats just now and everything looked great. Run a manual scan now or wait for Jetpack to scan your site later today. 

**After**
You resolved the 2 threats listed below and your site is now risk free.. To review your site again run a manual scan, or wait for Jetpack to scan your site later today.

### My Questions and Answers 😄 
- Does the distinction between scenario 2 and 3 exist in code?
- In general the 100% the new copy provided by editorial is much better and clearer. But I can see that the string saying **when** the last scan happened is useful. Particularly, in scenario 2 (user may think "great everything is fine, but as of **when**")
- I may have not been clear in my copy request that the time was a string, not just a generic "just now" which was not great language.
- I can ping the editor for review (can't find them on GitHub). In the meantime lets keep the string in there for now.

Reply:
@osullivanchris 

> You resolved 2 threats just now and everything looked great. Run a manual scan now or wait for Jetpack to scan your site later today.

This message 👆does not exist in the app after our recent tweaks to the fix flow. 

The app now shows below message while threats are fixing: 

> We\'re hard at work fixing these threats in the background. In the meantime feel free to continue to use your site as normal, you can check the progress at anytime.

and then (if threats are found)

> The scan found %1s potential threats with %2s. Please review them below and take action or tap the fix all button. We are %3$s if you need us.

or (if no threats are found)

> The last Jetpack scan ran %s and did not find any risks.

---------------------------------------------

>  In the meantime lets keep the string in there for now.

I'll go ahead and merge this PR. I'll create a new one once we hear from the editorial and have any changes.